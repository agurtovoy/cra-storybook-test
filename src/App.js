import React from 'react';


import logo from './logo.svg';
import Link from './components/link';

import './App.css';

import { css } from 'styled-components/macro';

const styles = {
    root: css`
      text-align: center;
    `,

    header: css`
        background-color: #282c34;
        min-height: 100vh;
        display: flex;
        flex-direction: column;
        align-items: center;
        justify-content: center;
        font-size: calc(10px + 2vmin);
        color: white;
    `,
}


function App() {
  return (
    <div css={styles.root}>
      <header css={styles.header}>
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <Link />
      </header>
    </div>
  );
}

export default App;
