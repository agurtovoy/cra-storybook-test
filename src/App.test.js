import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

import 'jest-styled-components';


test('renders learn react link', () => {
  const { getByText } = render(<App />);
  const linkElement = getByText(/learn react/i);
  expect(linkElement).toMatchSnapshot();
});
