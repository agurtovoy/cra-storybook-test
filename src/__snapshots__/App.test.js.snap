// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
.c0 {
  color: #61dafb;
}

<a
  class="c0"
  href="https://reactjs.org"
  rel="noopener noreferrer"
  target="_blank"
>
  Learn React
</a>
`;
